.github/workflows/blackhole-nightly-tests-20-cores-impl.yaml

```
@@ -0,0 +1,80 @@
+name: "[internal] Blackhole nightly tests 20 cores impl"
+
+on:
+  workflow_call:
+    inputs:
+      build-artifact-name:
+        required: true
+        type: string
+      wheel-artifact-name:
+        required: true
+        type: string
+      docker-image:
+        required: true
+        type: string
+      runner-label:
+        required: true
+        type: string
+        default: ''
+        description: '["P100"] (P100 only), ["P150"] (P150 only)'
+      arch_name:
+        required: false
+        type: string
+        default: 'blackhole'
+        description: 'Architecture name to use'
+
+jobs:
+  nightly-bh-models:
+    strategy:
+      # Do not fail-fast because we need to ensure all tests go to completion
+      # so we try not to get hanging machines
+      fail-fast: false
+      matrix:
+        test-group: [
+            { model: panoptic-deeplab, owner_id: U08ARDXARPF, cmd: pytest models/experimental/panoptic_deeplab/tests/pcc/ }, # Ilija Anastasijevic
+          ]
+    name: Nightly 20 cores ${{ matrix.test-group.model }}
+    # CIv2 to get weights
+    runs-on: ${{ format('tt-ubuntu-2204-{0}-stable', inputs.runner-label) }}
+    container:
+      image: ${{ inputs.docker-image }}
```

Switch to CIv2 docker proxy

```suggestion
      image: harbor.ci.tenstorrent.net/${{ inputs.docker-image || 'docker-image-unresolved!'}}
```